docs/content/cli-commands/npm-install.md

```diff
@@ -30,9 +30,16 @@ common options: [-P|--save-prod|-D|--save-dev|-O|--save-optional] [-E|--save-exa
 ### Description
 
 This command installs a package, and any packages that it depends on. If the
-package has a package-lock or shrinkwrap file, the installation of dependencies
-will be driven by that, with an `npm-shrinkwrap.json` taking precedence if both
-files exist. See [package-lock.json](/configuring-npm/package-lock-json) and [`npm shrinkwrap`](/cli-commands/shrinkwrap).
+package has a package-lock, lockfile, or shrinkwrap file, the installation of
```

Comment:
what is a "lockfile" that's not a package-lock or a shrinkwrap? It might be clearer to be explicit about the relevant filenames (i see you list them below, but that doesn't stop me from being confused when i read this paragraph)

Reply:
good point! 👍 thanks for the review 😊